src/ImGui.NET/Generated/ImGui.gen.cs

```diff
@@ -8085,297 +8054,6 @@ public static bool InputScalarN(string label, ImGuiDataType data_type, IntPtr p_
             }
             return ret != 0;
         }
-        public static bool InputTextWithHint(string label, string hint, string buf, uint buf_size)
```

Comment:
Why was this removed?

Reply:
Issue https://github.com/mellinoe/ImGui.NET/issues/215 shows that the **ImGui.gen** default implementation of the `InputTextWithHint`  feature is basically useless. This commit (https://github.com/mellinoe/ImGui.NET/commit/66f1afa9a268c2b70ff9403c570e45c8847e37a4) fixes that issue by manually adding public APIs for `InputTextWithHint`  feature in **ImGui.Manual.cs** class.

With that being said, in my opinion, in order to not confuse the users of this lib, it would be nice to remove the default implementation of `InputTextWithHint`  feature from  **ImGui.gen** class. In order to do that, I added this line (https://github.com/mellinoe/ImGui.NET/pull/221/files#diff-c5973b39c150ae89c1170e041fddade4da7998b0d994fad05394f4b311e402a9R104) in this MR and run the generator tool.

Let me know if you disagree with this thinking process and I will revert this part of the MR.

Reply:
I forgot that the manual overloads had already been merged in for TextWithHint. That clear it up, thanks.